app/assets/javascripts/admin/addon/templates/badges-award.hbs

```diff
@@ -21,6 +21,12 @@
           {{input type="checkbox" checked=replaceBadgeOwners}}
           {{i18n "admin.badges.mass_award.replace_owners"}}
         </label>
+        {{#if model.multiple_grant}}
+          <label class="grant-existing-holders">
+            {{input type="checkbox" checked=grantExistingHolders class="grant-existing-holders"}}
```

Comment:
I'm wonder if we'll run into styling problems if both label and input has the same class.